build.sbt

```diff
@@ -48,18 +48,11 @@ lazy val chronoscala = (project in file("."))
     }
   )
   .settings({
-    // Exclude 0.1.0 because 0.1.0 haven't released for Scala 2.12 .
-    // Fix if minor version is bumped.
-    val previousVersions = Set(1, 2, 3, 4, 5).map(patch => s"0.1.$patch")
+    val previousVersions = Set.empty
```

Comment:
This PR breaks binary compatibility.
I will bump minor version to `0.2.0` in next release.